src/components/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './HeaderStyles.css';
+
+const Header = () => (
+  <header className="headsection">
+    <h1 className="heading">
+      My Math Magician
+    </h1>
+    <ul className="lists">
+      <a href="/" className="list-item">Home</a>
+      <a href="/Calculator" className="list-item">Calculator</a>
```

Comment:
[OPTIONAL] I would highly suggest reading more about [Links in React](https://v5.reactrouter.com/web/api/Link) instead of using `anchor tag` that refreshes the page

Reply:
> [OPTIONAL] I would highly suggest reading more about [Links in React](https://v5.reactrouter.com/web/api/Link) instead of using `anchor tag` that refreshes the page

Noted, I'll be sure to look into that. Thank you!